src/App.tsx

```diff
@@ -1,7 +1,28 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+export const App: React.FC = () => {
+  const [selectedKey, setSelectedKey] = useState('');
+
+  const handleKey = (event: KeyboardEvent) => {
+    setSelectedKey(event.key);
+  };
+
+  useEffect(() => {
+    document.addEventListener('keyup', handleKey);
+
+    return () => document.removeEventListener('keyup', handleKey);
+  }, []);
+
+  return (
+    <div className="App">
+      <p className="App__message">
+
+        {selectedKey ? (
+          `The last pressed key is [${selectedKey}]`
+        ) : (
+          'Nothing was pressed yet'
+        )}
```

Comment:
you can keep it simple here
```suggestion
    {selectedKey 
      ? `The last pressed key is [${selectedKey}]`
      : 'Nothing was pressed yet'}
```